weaviate/batch/objectsCreate.go

```diff
@@ -17,8 +18,9 @@ type ObjectsBatchRequestBody struct {
 
 // ObjectsBatcher builder to add multiple objects in one batch
 type ObjectsBatcher struct {
-	connection *connection.Connection
-	objects    []*models.Object
+	connection       *connection.Connection
+	objects          []*models.Object
+	consistencyLevel string
```

Comment:
Would it make sense to use constants instead of strings?

Reply:
We have constants already in the server code, but they are not available in weaviate server version `1.17.x`. I can just upgrade to latest for now, until v1.18 is released

Reply:
actually, I forgot that we already had consistency level constants in this codebase. Updated to use those